src/cljc/athens/common_events.cljc

```diff
@@ -277,6 +277,34 @@
                      :new-uid new-uid}}))
 
 
+(defn unlinked-references-link
```

Comment:
So far we've used convention to start event builders with `build-*`, please continue doing it